crates/iota-data-ingestion-core/src/reducer.rs

```diff
@@ -37,11 +38,7 @@ pub trait Reducer<Mapper: Worker>: Send + Sync {
     /// determined by [`should_close_batch`](Reducer::should_close_batch). The
     /// implementation should handle the batch processing logic and return
     /// an error if the operation fails.
-    ///
-    /// # Note
```

Comment:
This was probably not intended, I think we do not want to remove this note.

Reply:
fixed: https://github.com/iotaledger/iota/pull/5820/commits/5f8bf926f4f91e0fe0539b47d415dce5b4b9fa8d#diff-3665ba807b5937734070baa5b4c12186d05842a235eb9b5422acd2a87c4cbe63R42